test/e2e-olm/utils.go

```diff
@@ -35,27 +37,47 @@ const (
 	exampleSecretsPath   = "/tmp/example-secrets"
 )
 
-func registerSchemes(t *testing.T) {
-	elasticsearchList := &loggingv1.ElasticsearchList{}
-	err := test.AddToFrameworkScheme(loggingv1.SchemeBuilder.AddToScheme, elasticsearchList)
+func setupK8sClient(t *testing.T) {
+	if _, ok := os.LookupEnv(TestOperatorNamespaceEnv); !ok {
+		t.Fatal("TEST_OPERATOR_NAMESPACE is unset")
+	}
+	operatorNamespace = os.Getenv(TestOperatorNamespaceEnv)
+	t.Logf("Found namespace: %q", operatorNamespace)
```

Comment:
nit. The typical usage for checking if an env exists is instead of using `os.LookupEnv`:
```
operatorNamespace := os.Getenv(TestOperatorNamespaceEnv)
if operatorNamespace == "" {
    t.Fatal("TEST_OPERATOR_NAMESPACE is unset")
}
```